crates/disks/Cargo.toml

```diff
@@ -26,4 +26,5 @@ proc-mounts = { path = "../proc-mounts" }
 rand = "0.5.5"
 rayon = "1.0.2"
 sys-mount = "1.0.3"
+sysfs-class = { git = "https://github.com/pop-os/sysfs-class" }
```

Comment:
I published sysfs-class. Let me know if you want to publish as well

Reply:
Will need to have it updated before I can use it here.

Reply:
Ok, it is published as version 0.1.1